src/api/authentication.js

```diff
@@ -10,6 +15,16 @@ export const signIn = async (payload) => {
   return req.ok;
 };
 
+// payload = {email, password}
+export const signInPassword = async (payload) => {
+  const req = await fetch(`${API_HOST}/login-password`, {
+    ...DEFAULT_POST,
+    body: JSON.stringify(payload),
+  }).then(checkSeeOther);
```

Comment:
Is it necessary? what does it catch? Receiving 200 will trigger an error?

Reply:
it checks that the response has the link attached to it. 
receiving an empty response will trigger an error